src/concrete_convex_hull.jl

```diff
@@ -398,11 +365,11 @@ For further details see
 function monotone_chain!(points::Vector{VN}; sort::Bool=true
                         )::Vector{VN} where {N<:Real, VN<:AbstractVector{N}}
 
-    @inline function build_hull!(semihull, iterator, points, zero_N)
+    @inline function build_hull!(semihull, iterator, points)
         @inbounds for i in iterator
             while length(semihull) >= 2 &&
                     (right_turn(semihull[end-1], semihull[end], points[i])
-                         <= zero_N)
+                         <= zero(N))
```

Comment:
i wonder if the `=` is needed or we can just we `!is_right_turn(...)`

Reply:
Mathematically the `=` is needed. In practice, with floating-point numbers it might not be needed, but if we have rationals, it is needed again.